### ISSUE TITLE
How can adjust custom layout height accoring to text length.

### ISSUE THREAD

**Issue by Gopinatha-L** [skydoves/Balloon#91]
**Please complete the following information:**
- Library Version - latest
- Affected Device(s) NA
 
**Describe the Bug:**

I"m using a custom layout that would contain a two text view's and it grows according to text length. I'm setting the text after the balllon was created by default height & width. 

**Expected Behavior:**

I want a dynamic height for ballon based on text size or context size in custom layout.

**Comment by skydoves** [comment]
Hi, could you test using parameter `maxWidth` to your TextViews?
```kotlin
val myTextView = customProfileBalloon.getContentView().findViewById<TextView>(R.id.myTextView)
myTextView.maxWidth = customProfileBalloon.getMeasureWidth()
```

**Comment by skydoves** [comment]
This measuring width & height size of the TextViews function will be included in the next release.

**Comment by Gopinatha-L** [comment]
Thanks !. I can use Width-ratio for my use case. But I'm interested in height of text-view in custom layout.

**Comment by Gopinatha-L** [comment]
Please make sure that text-view dynamic size applies for custom layout as well.

**Comment by skydoves** [comment]
#71

**Comment by Gopinatha-L** [comment]
Yep. It works. Thanks

**Comment by skydoves** [comment]
Hi, @Gopinatha-L!
It is released in a new version [1.2.2](https://github.com/skydoves/Balloon/releases/tag/1.2.2).

Added measureTextWidth function for measuring the exact text size in the custom layout.
measures the width of a TextView and set the measured with. If the width of the parent XML layout is wrapped content, and also the widths of TextViews in the parent layout is wrapped content, this functionality will measure the width exactly.
We can use this method below way.

```kotlin
val myTextView = customProfileBalloon.getContentView().findViewById<TextView>(R.id.myTextView)
customProfileBalloon.measureTextWidth(myTextView)
```

 Thanks for your issue!
